core-plugin/src/com/google/gct/idea/debugger/CloudDebugProcess.java

```diff
@@ -363,6 +364,26 @@ public void registerAdditionalActions(@NotNull DefaultActionGroup leftToolbar,
       }
     }
 
+    // remove help button since it points to the IntelliJ help by default and we don't have
+    // a help page yet.
+    // for some reason, the help button's key in leftToolbar is null, so we need to remove it
+    // by class name.
+    // https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/149
+    for (AnAction child : leftToolbar.getChildActionsOrStubs()) {
+      if (child.getClass().getCanonicalName().equalsIgnoreCase(
+          "com.intellij.ide.actions.ContextHelpAction")) {
+        // we never want to show IDEA's help.
+        leftToolbar.remove(child);
+
+        // show our help if we have it.
+        String helpUrl = GctBundle.getString("clouddebug.helpurl");
+        if (!helpUrl.equalsIgnoreCase("")) {
```

Comment:
!"".equals(helpUrl)

to avoid NullPointerExceptions and because we don't need to check for case in an empty string